Common/Data/Custom/Quiver/QuiverWallStreetBets.cs

```diff
@@ -0,0 +1,139 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using QuantConnect.Util;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+using NodaTime;
+using ProtoBuf;
+using static QuantConnect.StringExtensions;
+
+namespace QuantConnect.Data.Custom.Quiver
+{
+    /// <summary>
+    /// Mentions of the given company's ticker in the WallStreetBets daily discussion thread
+    /// </summary>
+    [ProtoContract(SkipConstructor = true)]
+    public class QuiverWallStreetBets : BaseData
+    {
+
+        /// <summary>
+        /// Date of the daily discussion thread
+        /// </summary>
+        [ProtoMember(10)]
+        [JsonProperty(PropertyName = "Date")]
+        [JsonConverter(typeof(DateTimeJsonConverter), "yyyy-MM-dd")]
+        public DateTime Date { get; set; }
+
+
+
+        /// <summary>
+        /// The follower count on the given date
+        /// </summary>
+        [ProtoMember(11)]
+        [JsonProperty(PropertyName = "Count")]
+        public int? Mentions { get; set; }
```

Comment:
Curious, why have this value be nullable? We could represent no mentions with `0`, right?

Also, I don't know if we'll want to filter some tickers out like `$ROPE`, since they might be a bit chaotic (maybe with single letter tickers too, depending on the signal to noise ratio). -> Let's have a talk about this on Slack 👍 

In the future, it'd be nice to have the contents of WSB posts for NLP parsing, but I'm very thankful as is that you're integrating this data set with us 😃